src/scripts/main.js

```diff
@@ -1,3 +1,330 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const table = document.querySelector('table');
+const tBody = table.querySelector('tbody');
+const tHeadCols = Array.from(table.tHead.rows[0].cells);
+const tRows = Array.from(table.rows);
+const tFootCols = Array.from(table.tFoot.rows[0].cells);
+const tBodyRows = table.tBodies[0].rows;
+
+function salaryToNum(str) {
+  const arr = str.split(',');
+  const value = arr[0].slice(1) + arr[1];
```

Comment:
The `salaryToNum` function assumes that the salary string will always have a comma. This might lead to errors if the salary format changes or if the input is not as expected. Consider adding a check to handle cases where the salary string might not contain a comma.